env/includes/desktop/project.yml

```diff
@@ -49,6 +49,7 @@ desktop.project:
     - '*Motion*'
     - '*Nuke*'
     - '*Photoshop*'
+    - '*After*'
```

Comment:
`tk-config-default2` had an '*Effects*' entry too - can you explain a little more around this setup?

Reply:
This is just to move After Effects inside the `Creative Tools` section. I added "Effects" to be in line with `tk-config-default2`